doc/ref_model/chapters/chapter04.md

```diff
@@ -21,25 +26,139 @@ The idea of the infrastructure profiles catalogue is to have a predefined set of
 
-| .conf | vCPU | RAM | Local Disk | Management Interface |
-|-----------|------|-------|------------|----------------------|
-| .tiny | 1 | 512MB | 1GB | 1Gbps |
-| .small | 1 | 2GB | 40GB | 1Gbps |
-| .meduim | 2 | 4GB | 40GB | 1Gbps |
-| .large | 4 | 8GB | 80GB | 1Gbps |
-| .large2* | 4 | 16GB | 80GB | 1Gbps |
-| .xlarge* | 8 | 16GB | 160GB | 1Gbps |
-| .xlarge2* | 8 | 32GB | 160GB | 1Gbps |
-| .xlarge3* | 8 | 64GB | 160GB | 1Gbps |
+Flavours represent the compute, memory, storage capacity, and management network resource templates that are used to create the VMs on the compute hosts. Each VM instance is given a flavour (resource template), which determines the instance’s core, memory and storage characteristics. 
 
-<p align="center"><b>Table 4-1:</b> Compute flavours.</p>
+Flavours can also specify secondary ephemeral storage, swap disk, metadata to restrict usage, or special project access. In other terms, Flavours are grouping of hosts typically sharing same characteristics or metadata. 
+
+A compute flavour geometry consists of the following elements:
+
+Element |Description 
+--------|----------
+Name	|A descriptive name
+Virtual compute resources (aka vCPUs) |Number of virtual compute resources (vCPUs) presented to the instance.
+Memory MB	|Instance memory in megabytes. 
+Ephemeral/Local Disk |Specifies the size of an ephemeral data disk that exists only for the life of the instance. Default value is 0.<br />The ephemeral disk may be partitioned into boot (base image) and swap space disks. 
+Is Public	|Boolean value, whether flavor is available to all users or private to the project it was created in. Defaults to True.
+Extra Specs	|Key and value pairs that define on which compute nodes a flavor can run. These pairs must match corresponding pairs on the compute nodes. Use to implement special resources, such as flavors that run on only compute nodes with GPU hardware.
+
+<p align="center"><b>Table 4-1:</b> Flavour Geometry Specification.</p>
+
+Flavour Capabilities:
+
+1. **CPU Oversubscription Ratio**: is based on the number of threads available. For example, on a 24-core host with HT, there are 48 vCPUs with 1:1 CPU Ratio and 96 vCPUs with 2:1 CPU Ratio.
+
+2. **HT (Hyper Threading support)**: Enabled on all servers. Gets 2 hyper threads per physical CPU.  Always ON. 
+3. **CPU Pinning**: vCPU is pinned to a physical core and dedicated to the requesting VM. 
+4. **NUMA (Non-Uniform Memory Access) Alignment**: Indicates that vCPU will be on a Socket that is aligned with the associated NIC card and memory.  Important for performance optimized VNFs. 
+5. **Huge Pages**: By default, CPU allocate RAM in 4K chunks. Huge Pages enable to allocate in larger Chunks (such as 2MB, 1GB).  This helps improve performance in some cases. 
+6. **SR-IOV (Single-Root Input/Output Virtualization)**: Allows SR-IOV ports to be used in VM. 
```

Comment:
Maybe it is better to explain as Direct Transfer which is kernel bypass technologies , SR-IOV and PCI Pass. 

**Direct Transfer:** This is the fastest option for handling packet in the user data plane. There are two options available which are SR-IOV and PCI pass-through, SR-IOV can virtualize the NIC to several VM to use but PCI pass-through will cause NIC to be available only for one VM.

Reply:
Great , but we initially  agreed that SRIOV and PCI-PT not our focus in this document , 
also small comment on the definition "SR-IOV can virtualize the NIC to several VM to use but PCI pass-through will cause NIC to be available only for one VM."

we can define it SR-IOV can virtualize the NIC to several VF not VM

Reply:
Agree with Ahmed. This PR has been brought directly from the original CR. We need to make sure it is aligned with the principles we defined in chapter 1. I will take a stab at editing it in a way that I believe will align with our principles and bring it for review. Is that OK Burak?

Reply:
it’s ok @rabiabdel